src/cf-core/src/cf-utilities.less

```diff
@@ -33,14 +34,14 @@
 //
 
 .u-visually-hidden {
-  position: absolute;
-  width: 1px;
-  height: 1px;
-  border: 0;
-  margin: -1px;
-  padding: 0;
-  overflow: hidden;
-  clip: rect(0 0 0 0);
+    position: absolute;
+    width: 1px;
+    height: 1px;
+    border: 0;
+    margin: -1px;
+    padding: 0;
+    overflow: hidden;
+    clip: rect(0 0 0 0);
```

Comment:
Then can you please add a comment that commas aren't being used for backwards-compatibility?

I know that screenreader-only utilities like these like to have their bases covered with more CSS than is necessary to make sure it doesn't render for sighted users, and I'm all in favor of that.